logger.ts

```diff
@@ -0,0 +1,36 @@
+import fetch from 'cross-fetch';
+const _debounce = require('lodash/debounce');
+
+const ingestEndpoint = process.env.AXIOM_INGEST_ENDPOINT || process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT;
+
+const debouncedSendLogs = _debounce(() => sendLogs(), 1000);
+
+let collectedLogs: any[] = [];
+
+function _log(level: string, message: string, args: any = {}) {
+  const l = { level, message, ...args };
+  // TODO: print to console?
+  console.log(l);
```

Comment:
Very good question. I think we should not do it at this point, if we get the feedback that people want this, we can add it, maybe as an option.

Reply:
removed